bundles/pubsub/pubsub_serializer_avrobin/src/pubsub_avrobin_serialization_provider.c

```diff
@@ -0,0 +1,120 @@
+/**
+ *Licensed to the Apache Software Foundation (ASF) under one
+ *or more contributor license agreements.  See the NOTICE file
+ *distributed with this work for additional information
+ *regarding copyright ownership.  The ASF licenses this file
+ *to you under the Apache License, Version 2.0 (the
+ *"License"); you may not use this file except in compliance
+ *with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+#include "pubsub_avrobin_serialization_provider.h"
+
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+#include <assert.h>
+
+#include "avrobin_serializer.h"
+#include "dyn_message.h"
+#include "celix_log_helper.h"
+#include "pubsub_message_serialization_service.h"
+
+static void dfi_log(void *handle, int level, const char *file, int line, const char *msg, ...) {
+    va_list ap;
+    celix_log_helper_t *log = handle;
+    char *logStr = NULL;
+    va_start(ap, msg);
+    vasprintf(&logStr, msg, ap);
+    va_end(ap);
+    celix_logHelper_log(log, level, "FILE:%s, LINE:%i, MSG:%s", file, line, logStr);
+    free(logStr);
+}
+
+
+static celix_status_t pubsub_avrobinSerializationProvider_serialize(pubsub_serialization_entry_t* entry, const void* msg, struct iovec** output, size_t* outputIovLen) {
```

Comment:
Why outputIovLen? Not even used?

Reply:
Yes, for the current serialization (avrobin/json) outputIovLen is not used